src/proxy/cpsubsystem/AtomicLongProxy.ts

```diff
@@ -45,8 +45,7 @@ export class AtomicLongProxy extends BaseCPProxy implements IAtomicLong {
         }
         return this.encodeInvokeOnRandomTarget(AtomicLongAddAndGetCodec, this.groupId, this.objectName, delta)
             .then((clientMessage) => {
```

Comment:
Minor: Can be replaced with `.then(AtomicLongAddAndGetCodec.decodeResponse)`

Reply:
Good catch. Fixed in 1ce0e2d1eaedae13ece036ae6db2089cfa6dd745